action.yml

```diff
@@ -173,6 +181,7 @@ runs:
       run: |
         echo "pre-check reset"
         git reset --hard && git clean -fd
+        cd ${{ inputs.project_path }}
```

Comment:
it's probably better to do something like `$( cd ${{ inputs.project_path }} && lekko bisync )` but I'm not confident in my knowledge about subshells.